### DISCUSSION TITLE
Can I backup and restore ReplicatedMergeThree tables?

### DISCUSSION THREAD

**Discussion by Programmeris** [Altinity/clickhouse-backup#632] [Q&A] [answered]
From the documentation, I still did not understand whether it is possible to backup and restore ReplicatedMergeThree tables without problems. This is very important in my case.

**Comment by nikola358** [comment, marked as answer]
Yes you can. You can use the clickhouse-backup-restore from https://github.com/AlexAkulov/clickhouse-backup/blob/master/Examples.md to restore them. I tested it with a clickhouse cluster with 3 shards and 2 replicas. After a shard is restored the data will be replicated as one would expect.

**Reply by Programmeris** [reply]
Thanks for answer!
